### DISCUSSION TITLE
What is the difference between observable counter and guage?

### DISCUSSION THREAD

**Discussion by aaron-ai** [open-telemetry/opentelemetry-java#4594] [Q&A] [answered]
I found there is an`ObservableLongCounter/ObservableDoubleCounter` except the gauge, which makes me feel confused. So what is the difference?

**Comment by jkwatson** [comment, marked as answer]
Please see the metrics specification for this: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md

**Reply by aaron-ai** [reply]
Got it. Thanks for reply!
